### ISSUE TITLE
fix: incompatible version between natds-rn and natds-styles 

### ISSUE THREAD

**Issue by yasminnvaz** [natura-cosmeticos/natds-rn#63]
**Describe the bug**

I started a new react-native project, using the following versions:
- natds-rn: `2.0.1`
- natds-styles: `0.17.0`

When I started the project, applying a button component and using `yarn ios` or `yarn android`, the error bellow appeared: 
`Cannot read property 'colorPrimaryLight' of undefined`

**To Reproduce**
1. Install the latest version of `natds-rn` and `natds-styles`
2. Apply any component from design system
3. Start your application using `yarn ios` or `yarn android`

**Expected behavior**
- See my component working correctly

**Screenshots**
![image](https://user-images.githubusercontent.com/42593623/89676981-f5f5c080-d8c2-11ea-9ae7-7350ad995a15.png)

**Environment (please complete the following information):**
- natds-rn: `2.0.1`
- natds-styles: `0.17.0`

Changing the versions to 2.0.0 and 0.5.1, respectively, works

**Comment by jimmyandrade** [comment]
@yasminnvaz 

Thank you for using the Design System. We are sorry that you experienced this problem when using our library.

Thanks to your issue, we detected a problem that causes versions `0.17.0` to `0.17.4` of `natds-styles` to present errors during installation or update, especially in new projects.

Therefore, we would like you to try to use version 0.17.5 (or higher) of `natds-styles` to see if the problem continues.

**Comment by yasminnvaz** [comment]
Hi, @jimmyandrade!

Thank you for your answer.
I tried to follow these steps but it didn't work, even after clearing the cache. The same error appears :(

**Comment by jimmyandrade** [comment]
Hi, @yasminnvaz 

What version of `natds-styles` is your project using now?

**Comment by yasminnvaz** [comment]
Hi!

I followed your recommendation and tried using the version `0.17.5` of `natds-styles` with `0.2.1` of `natds-rn`

**Comment by jimmyandrade** [comment]
Hi @yasminnvaz 

Sorry for the delay. I asked @lauralucca for help, we're gonna call you :)

**Comment by yasminnvaz** [comment]
Hi, people!

@lauralucca and I realised that my problem was not because of the version, but because I passed three parameters (tokens, brand, mode) to `buildTheme` instead of two (brand, mode).

Therefore, I want to suggest that we update the [Setup Theme](https://github.com/natura-cosmeticos/natds-rn#setup-the-theme) in the `README` because it suggests that we pass three parameters but if we look at the function it only accepts two.

Thank you for all the help, @jimmyandrade and @lauralucca <3
